### PR TITLE
NFA extra test

### PR DESCRIPTION
**Pull request by eliotwrobson**
@caleb531 Adds some extra tests for regex parsing to NFAs and style in DFA minimization.

### CODE REVIEW

File: automata/fa/nfa.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def _from_symbol(cls, symbol, input_symbols=None):
     @classmethod
     def from_regex(cls, regex):
         """Initialize this NFA as one equivalent to the given regular expression"""
-        input_symbols = set(regex) - {'*', '|', '(', ')', '?'}
+        input_symbols = set(regex) - {'*', '|', '(', ')', '?', ' ', '\t'}
```
Comment (caleb531) [inline automata/fa/nfa.py:126]: @eliotwrobson What is the purpose of the extra regex symbols?

Comment (eliotwrobson) [reply automata/fa/nfa.py:126]: Allows regex to ignore whitespace. The lexer is already configured to skip whitespace by default, without this change regexes will have whitespace passed in as input symbols.

Comment (caleb531) [reply automata/fa/nfa.py:126]: @eliotwrobson Is whitespace not significant in a automata theory regex (like it is in Perl regex)?

Comment (eliotwrobson) [reply automata/fa/nfa.py:126]: The way it's configured now it's just ignored. You can't use whitespace as implicit parentheses. 

This behavior can actually be changed but that's a little beyond the scope of this PR

Comment (caleb531) [reply automata/fa/nfa.py:126]: @eliotwrobson Makes sense. This looks good, then.

**Review by caleb531 — APPROVED** [review]

**Pull request merged**